q2_fmt/plugin_setup.py

```diff
@@ -360,4 +325,51 @@
     }
 )
 
+plugin.methods.register_function(
+    function=q2_fmt.peds_simulation,
+    inputs={'table': FeatureTable[Frequency | RelativeFrequency |
+                                  PresenceAbsence]},
+    parameters={'metadata': Metadata,
+                'time_column': Str,
+                'reference_column': Str,
+                'subject_column': T_subject,
+                'filter_missing_references': Bool,
+                'drop_incomplete_subjects': Bool,
+                'drop_incomplete_timepoint': List[Str],
```

Comment:
The `List[Str]` type doesn't align with the description above - reminder to sort that out.